it/base/ListYearsFeatureSwitchedISpecBase.scala

```diff
@@ -0,0 +1,97 @@
+package base
+
+import com.github.tomakehurst.wiremock.WireMockServer
+import com.github.tomakehurst.wiremock.client.WireMock.configureFor
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import org.scalatestplus.play.guice.GuiceOneServerPerSuite
+import play.api.Application
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.libs.ws.{DefaultWSCookie, WSClient, WSCookie}
+import play.api.mvc.{Cookie, Session, SessionCookieBaker}
+import uk.gov.hmrc.crypto.PlainText
+import uk.gov.hmrc.http.SessionKeys
+import uk.gov.hmrc.play.bootstrap.frontend.filters.crypto.SessionCookieCrypto
+
+trait ListYearsFeatureSwitchedISpecBase extends AnyWordSpec with Matchers with GuiceOneServerPerSuite with BeforeAndAfterAll with TestData {
+  sealed trait Language
+
+  case object English extends Language
+
+  case object Welsh extends Language
+
+  val mockHost = "localhost"
+  val mockPort = 11111
+
+  lazy val wireMockServer: WireMockServer = new WireMockServer(wireMockConfig().port(mockPort))
+
+  val mockedMicroservices: Set[String] = Set(
+    "property-linking",
+    "business-rates-authorisation",
+    "auth"
+  )
+
+  val config: Map[String, String] = Map(
```

Comment:
You probably don't need to duplicate the entire ISpecBase to change the config for the test you want.

You can try overriding the config val in the ISpec that needs to be modified with the values that you need for that test.